package.json

```diff
@@ -5,6 +5,8 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
+    "build:prod": "yarn init-appregistry && yarn build",
+    "init-appregistry": "cp src/assets/prod.appregistry.json src/assets/appregistry.json",
```

Comment:
Two comments about this:
1. You can use the `-n` ("no clobber") option to prevent overwriting a file that has already been copied here. Then we can include an `init-appregistry:force` that looks like this line (i.e., *does* overwrite the destination file). I just tested it, and when the file already exists, `cp` still exits with success. That would allow the `yarn build` script to work safely without having to split it into a `build` and a `build:prod` script.
2. It's a bit risky to use a Linux command line program here, since this won't work on e.g. Windows. Now, we don't really support Windows, so I'm ok with this at least for now, but there are :nauseated_face: NPM-based cross-platform solutions instead. I wouldn't look into any of those for now, but let's open an issue to track this possible future pitfall.

Reply:
1. I would say if we're going to use the `-n` flag (which we probably should) in order to just have one `build` script, then there's probably no need for an additional `init-appregistry:force` script. In my mind, if an `appregistry.json` is already there, then that satisfies the "initialize" condition (the condition being, an `appregistry.json` is required for the application to run).

2. Good catch, although I agree we don't really deal with windows at the current moment. I'd say in this case we could probably just punt on this until it actually becomes an issue, since even if it does eventually become an issue, a fix / one-time fix is low cost.

Reply:
Agree with both points. So, use `-n` and dump the force script, and don't worry about filing an issue for point (2) (since it will be obvious if and when it pops up).